spec/requests/api/v1/game_play_request_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+require './././lib/modules/serviceable'
+
+RSpec.describe "Game Play" do
+  context 'game play', :vcr do
+    it 'returns a photo, correct answer and group of choices' do
+      num = nil
+      get "/api/v1/game_play_apis?num=#{num}"
```

Comment:
Q: what happens if they do not pass in "num"? ... should we test what the response is in that scenario to affirm that our app responds appropriately?

Reply:
Completed, see [6a91437](https://github.com/Correct-or-Crass/Trivia_API/pull/41/commits/6a9143709aa093b4d47887bbc410df2b7f7f4543) below